examples/pipelines/df_fork.py

```diff
@@ -13,18 +13,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+from disdat.pipe import PipeTask
+import disdat.api as api
+from df_dup import DataMaker
+import luigi
+import pandas as pd
+
 """
 Fork Pipe
 
 Take an input DataFrame.  For each row, convert to a json string, and send to an upstream
 task specified by the string fork_task_str ('module.Class').
 
-author: Kenneth Yocum
+This examples shows:
+1.) How the input bundle shows up automatically as the parameter "pipeline_input" in upstream tasks.
+2.) How you can pass arguments when calling apply
+3.) And how an argument may be a string indicating an upstream task: '__main__.PrintRow'
+4.) How you may use **kwargs to find the dynamic set of inputs
+
+Pre Execution:
+$export PYTHONPATH=$DISDAT_HOME/disdat/examples/pipelines
+$dsdt context examples; dsdt switch examples
+$dsdt apply - DataMaker df_dup.DataMaker
+
+Execution:
+$python ./df_fork.py
+or:
+$dsdt apply DataMaker - df_fork.DFFork --fork_task df_fork.PrintRow
```

Comment:
```
$ dsdt apply DataMaker - df_fork.DFFork --fork_task df_fork.PrintRow
PrintRow[0]: <pandas.core.indexing._iLocIndexer object at 0x1101bff50>
PrintRow[3]: <pandas.core.indexing._iLocIndexer object at 0x11019ef90>
PrintRow[1]: <pandas.core.indexing._iLocIndexer object at 0x110144c90>
PrintRow[2]: <pandas.core.indexing._iLocIndexer object at 0x11013dfd0>
DFFork finished[task_0] with output[True]
DFFork finished[task_1] with output[True]
DFFork finished[task_2] with output[True]
DFFork finished[task_3] with output[True]
```

My guess is that this is actually supposed to print the contents of the dataframe, as opposed to the object references?

Reply:
oops.   df.iloc[], not df.iloc().  fixed.